noteEmulatorStartFix.txt

```diff
@@ -0,0 +1,14 @@
+----GENERAL INFORMATION AFTER EMULATOR-START GOAL FIX -----
+
+
+1)SET ANDROID_HOME environment variable and in the path %ANDROID_HOME%/tools and platform-tools
+
+2)User properties  _android.devices_  or _android.device_  maybe are not supported because will run always the emulator. 
```

Comment:
Because in my method to run android emulator I don't use the "AndroidDebugBridge" object that maybe return the devices set.